spec/features/login_visit_candidate_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+feature 'Visit Election' do
+  let(:user) { create(:user) }
+  let(:election) { create(:election,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.